contracts/meta-oracles/TimeSeriesFeed.sol

```diff
@@ -146,4 +158,34 @@ contract TimeSeriesFeed is
             _numDataPoints
         );
     }
+
+    /* ============ Public ============ */
+
+    /*
+     * Generate struct that holds TimeSeriesFeed's current nextAvailableUpdate, updateInterval,
+     * and previously logged prices.
+     *
+     * @returns                Struct containing the above params                  
+     */
+    function getTimeSeriesFeedState()
+        public
+        view
+        returns (TimeSeriesStateLibrary.State memory)
+    {
+        // Get timeSeriesData price values
+        uint256[] memory previousLoggedPrices;
+        if (timeSeriesData.dataArray.length == maxDataPoints) {
```

Comment:
Can't you just do the following without the if/else statement?
```
previousLoggedPrices = readList(timeSeriesData, timeSeriesData.dataArray.length);
```

Reply:
+1